src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -69,7 +68,6 @@ public boolean equals(Object other) {
             return false;
         }
 
-        // state check
         PersonCard card = (PersonCard) other;
         return id.getText().equals(card.id.getText())
```

Comment:
Should we add a comment here to inform the reviewer that the 'Label equals()', the id , behavior differs from our 'equals()' method?

Reply:
@Eclipse-Dominator your thoughts?

Reply:
> @Eclipse-Dominator your thoughts?

@Eclipse-Dominator reminder ...

Reply:
This method is currently not used in AB3 and does not really fit the idea of `equal`, thus it's better off removed from the source code. 

I have included the commit from AB4 that removes this method.